source/docs/casper/resources/build-on-casper/casper-env-prerequisites.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Casper Network development environment prerequisites
+---
+
+# Casper Network development environment prerequisites
+
+To be able to develop comfortably on the Casper Network you should use either `macOS` or `Linux Ubuntu 20.04`. Developing on Windows is not advised.
+Follow the steps below to install necessary software for the development environment.
+
+## Install the environment on macOS {#install-macos}
+
+### Install xcode command line tools {#install-xcode}
+
+```bash
+xcode-select --install
+```
+
+Check the version 
+
+```bash
+xcode-select -p
+```
+
+### Install rust {#install-rust}
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Check the rustup version
+
+```bash
+rustup --version
+```
+
+### Install brew {#install-brew}
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+### Install additional packages to be able to install cargo-casper and casper-client {#install-adds}
+
+```bash
+brew install pkg-config
+brew install openssl 
+```
+
+### Install the cargo-casper {#install-cargo-casper}
+
+```bash
+cargo install cargo-casper
+```
+
+Check the version
+
+```bash
+cargo-casper --version
+```
+
+### Install the casper client {#install-casper-client}
+
+```bash
+cargo install casper-client
+```
+
+Check the version
+
+```bash
+casper-client --version
+```
+
+For more information about supported commands, you can check it with:
+
+```bash
+casper-client --help
+```
+
+:::tip
+
+Optionally if you are going to run a Casper Network validator install cmake with the command below:
+
+:::
+
+### Install cmake {#install-cmake}
+
+```bash
+brew install cmake
+```
+
+Check the version of cmake
+```bash
+cmake --version
+```
+
+With all the above steps you should be able to succesfully go through the examples and tutorials in the documentation.
+
+## Install the environment on Linux {#install-linux}
+
+### Get curl {#install-curl}
+
+```bash
+sudo apt install curl
+```
+
+### Install rust {#install-linux-rust}
```

Comment:
Maybe instead link to the getting-started with rust? Also, in both the macos and linux setup rust-nightly is missing. Without rust-nightly some builds may fail.

Reply:
The content will be moved to the dev prerequisites in the Developers section. We will see how it looks and then we will add nightly where needed.